include/battle.h

```diff
@@ -802,7 +802,7 @@ struct BattleStruct
     u8 categoryOverride; // for Z-Moves and Max Moves
     u32 stellarBoostFlags[NUM_BATTLE_SIDES]; // stored as a bitfield of flags for all types for each side
     u8 fickleBeamBoosted:1;
-    u8 obedienceResult:3;
+    u8 padding:6;
```

Comment:
Any reason we don't just move the padding to the end? It'd make it more clear where a new u8 starts

Reply:
I would rather do that in upcoming to avoid conflicts.

Reply:
actually I did it right now.